pinot-spi/src/main/java/org/apache/pinot/spi/stream/StreamConfigProperties.java

```diff
@@ -45,6 +45,7 @@ private StreamConfigProperties() {
   public static final String STREAM_CONNECTION_TIMEOUT_MILLIS = "connection.timeout.millis";
   public static final String STREAM_IDLE_TIMEOUT_MILLIS = "idle.timeout.millis";
   public static final String STREAM_DECODER_CLASS = "decoder.class.name";
+  public static final String STREAM_DECODER_ERRORS_IGNORE = "decoder.errors.ignore";
```

Comment:
(minor) I feel it will be more nature this way since we changed it to boolean. Same for other places
```suggestion
  public static final String STREAM_DECODER_IGNORE_ERRORS = "decoder.ignore.errors";
```